### PR TITLE
Use Ubuntu20 for target nodes

### PR DESCRIPTION
**Pull request by fmuyassarov**
Switch to Ubuntu 20.04 LTS (Focal Fossa) for target nodes.

### CODE REVIEW

**Comment by Jaakko-Os** [comment]
lgtm

**Comment by fmuyassarov** [comment]
There was an issue with ironic-image. The [fix ](https://github.com/metal3-io/ironic-image/pull/189)is merged now. Re-triggering the CI

/test-integration
/test-centos-integration

File: vm-setup/roles/v1aX_integration_test/vars/main.yml [hunk 1]
```diff
@@ -56,8 +56,8 @@ IRONIC_IMAGE_DIR: "{{ lookup('env', 'IRONIC_IMAGE_DIR') | default('/opt/metal3-d
 IRONIC_ENDPOINT_BRIDGE: "{{ lookup('env', 'CLUSTER_PROVISIONING_INTERFACE') | default('provisioning')}}"
 
 # Distibution specific environment variables
-IMAGE_NAME: "{{ lookup('env', 'IMAGE_NAME') | default('bionic-server-cloudimg-amd64.img', true)}}"
-RAW_IMAGE_NAME: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('bionic-server-cloudimg-amd64-raw.img', true)}}"
+IMAGE_NAME: "{{ lookup('env', 'IMAGE_NAME') | default('focal-server-cloudimg-amd64.img', true)}}"
+RAW_IMAGE_NAME: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('focal-server-cloudimg-amd64-raw.img', true)}}"
 IMAGE_LOCATION: 'https://cloud-images.ubuntu.com/bionic/current'
```
Comment (furkatgofurov7) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:61]: Why are we leaving this as bionic?

Comment (fmuyassarov) [reply vm-setup/roles/v1aX_integration_test/vars/main.yml]: Forgot that. Thank you, fixed now.

**Comment by furkatgofurov7** [comment]
@fmuyassarov you should probably have to update README as well?

**Comment by fmuyassarov** [comment]
> @fmuyassarov you should probably have to update README as well?



> @fmuyassarov you should probably have to update README as well?

Done, thanks.

**Pull request merged**